pandas/tests/io/test_parquet.py

```diff
@@ -348,6 +348,20 @@ def test_cross_engine_fp_pa(df_cross_compat, pa, fp):
         tm.assert_frame_equal(result, df[["a", "d"]])
 
 
+def test_pyarrow_list():
+    pytest.importorskip("fastparquet")
+    import pyarrow as pa
+
+    list_int = pa.list_(pa.int64())
+    s = pd.Series([[1, 1], [2, 2]], dtype=pd.ArrowDtype(list_int))
+
+    df = pd.DataFrame(s, columns=["col"])
+    df.to_parquet("ex.parquet")
+
+    result = read_parquet(path="ex.parquet", dtype_backend="pyarrow")
```

Comment:
This seems off to me - so even with `dtype_backend="pyarrow"` we are trying to convert it to an `object` type? I think this should be maintain the pyarrow type all the way through

Reply:
Hi William.
I do agree with you and I might be wrong.
But at least for this specific case, the object type is preserved.
Nevertheless, I will try to work on a better solution.